man-roxygen/dots.R

```diff
@@ -1,3 +1,6 @@
-#' @param ... Additional arguments passed to an HTTP request function, such as
-#'   \code{\link[httr]{GET}}, \code{\link[httr]{POST}}, or
-#'   \code{\link[httr]{DELETE}}.
+#' @param ... Additional arguments passed to an HTTP request function,
+#'   such as \code{\link[httr]{GET}}, \code{\link[httr]{POST}}, or
+#'   \code{\link[httr]{DELETE}}. By default, HTTP requests use
+#'   values cached from previous identical calls. Use
+#'   \code{use_cache=FALSE} (or `Sys.setenv(DATAVERSE_USE_CACHE =
```

Comment:
Can we add `use_cache` as a documented argument in `get_file` as well? Currently it only shows up as a part of a the dot params arguments.  I want `use_cache` to be visible in `?get_dataframe_by_name`. That inherits the arguments of get_file via `@inheritDotParams get_file`  but that does not pull in the description of `cache`, so you don't see the cache explained in `?get_dataframe_by_name`